data/mods/DinoMod/mapgen/microlab_DinoLab.json

```diff
@@ -0,0 +1,1150 @@
+        { "chunks": [ "concrete_wall_ew" ], "x": 0, "y": 0, "neighbors": { "north": "microlab_rock_border" } },
+        { "chunks": [ "concrete_wall_ns" ], "x": 23, "y": 0, "neighbors": { "east": "microlab_rock_border" } },
+        { "chunks": [ "concrete_wall_ew" ], "x": 0, "y": 23, "neighbors": { "south": "microlab_rock_border" } },
+        { "chunks": [ "concrete_wall_ns" ], "x": 0, "y": 0, "neighbors": { "west": "microlab_rock_border" } }
+      ],
+      "palettes": [ "microlab" ]
+    }
+  },
+  {
+    "type": "mapgen",
+    "om_terrain": [ [ "microlab_DinoLab_edge" ] ],
+    "method": "json",
+    "object": {
+      "fill_ter": "t_strconc_floor",
+      "rows": [
+        "     | cc |  |    |     ",
+        "  6||||||||  |U  U|c hc ",
+        "  6|U     2YY|U  U|c hc ",
+        "  6|U ccci|  |(22(|h    ",
+        " 66|U     |  |      YY  ",
+        "||||U ccci|  |UUUUUU  ||",
+        " FF||2|||||22|||||||22| ",
+        "  c|  |U  |  2    F|  | ",
+        "  6|  |U  |  |c   F|  | ",
+        "  6|  |ch |  |ccc  |  | ",
+        "|5||  |c  |  |||(((|  ||",
+        "   2YY|ch |        2YY  ",
+        "   2YY|cc |        2YY  ",
+        "|2||  |||2|  |22(((|((2|",
+        "  c|  ( h (  (    6|h   ",
+        "  c|  (cc (  (ccYh6|66  ",
+        "  c|  (cc (  (ccY 6|66  ",
+        "  c|  ( h (  (   h6|h   ",
+        "|2||  |||2|  |    6|((2|",
+        "  U|       YY|22(((|6   ",
+        "  U|       YY|  666|c   ",
+        "  U|2|||2||22|   h |c   ",
+        "   | c|U U|  |^BBB^|U   ",
+        "   |      |  |||||||    "
+      ],
+      "place_nested": [
+        { "chunks": [ "concrete_wall_ew" ], "x": 0, "y": 0, "neighbors": { "north": "microlab_rock_border" } },
+        { "chunks": [ "concrete_wall_ns" ], "x": 23, "y": 0, "neighbors": { "east": "microlab_rock_border" } },
+        { "chunks": [ "concrete_wall_ew" ], "x": 0, "y": 23, "neighbors": { "south": "microlab_rock_border" } },
+        { "chunks": [ "concrete_wall_ns" ], "x": 0, "y": 0, "neighbors": { "west": "microlab_rock_border" } }
+      ],
+      "palettes": [ "microlab" ]
+    }
+  },
+  {
+    "type": "mapgen",
+    "om_terrain": [ [ "microlab_DinoLab_edge" ] ],
+    "method": "json",
+    "object": {
+      "fill_ter": "t_strconc_floor",
+      "rows": [
+        "     | cc |  |    |     ",
+        "  6||||||||  |U  U|c hc ",
+        "  6|U     2YY|U  U|c hc ",
+        "  6|U ccci|  |(22(|h    ",
+        " 66|U     |  |      YY  ",
+        "||||U ccci|  |UUUUUU  ||",
+        " FF||2|||||2||||||||||| ",
+        "  c|         2    FUUU  ",
+        "  6|^  hhh  ^|c   FU U  ",
+        "  6|^  ttt  ^|ccc       ",
+        "|5||   hh   h|||(((|||||",
+        "   2Y               YY  ",
+        "   2Y               YY  ",
+        "|2||=======[=|22(((|((2|",
+        "  c|?  t ? Y6|     c    ",
+        "  c|?  t ? Y^| cc       ",
+        "  c|?  t ?  ^| cc  c    ",
+        "  c|^       ^|     c    ",
+        "|2||v       ^|     |((2|",
```

Comment:
Ditto re `v`.

Reply:
Yeah, I've reconstructed the whole thing. Hopefully fixed now, kind of a pain porting mapgen stuff between CDDA and BN